src/config.py

```diff
@@ -6,15 +6,16 @@
 _path = os.path.dirname(os.path.abspath(__file__))
 _icon_path = _path + '/icons/'
 
+APP_ICON_NAMES = ["Colored Icon", "Light Icon", "Dark Icon"]
```

Comment:
Won't it be more suitable to hear if we call `Colored Icon` as `default`.

Reply:
I think you're right. I'm going to change it tomorrow. Do you prefer Default or Default Icon? I think the latter is better. As file name we could do default-icon.png. Also should I add the .svg file for the symbolic icons to the project. This could be useful if we need higher resolution one day.

Reply:
**Default Icon** is fine. 

> Also should I add the .svg file for the symbolic icons to the project.

That'll be great.